micropipenv.py

```diff
@@ -89,6 +90,11 @@
 _DEBUG = int(os.getenv("MICROPIPENV_DEBUG", 0))
 _NO_LOCKFILE_PRINT = int(os.getenv("MICROPIPENV_NO_LOCKFILE_PRINT", 0))
 _NO_LOCKFILE_WRITE = int(os.getenv("MICROPIPENV_NO_LOCKFILE_WRITE", 0))
+_FILE_METHOD_MAP = {  # The order here defines priorities
```

Comment:
The order in the dictionary is not guaranteed across all the Python interpreter versions, even though we test with Python 3.6+ where the order is guaranteed. It would be better to be explicit here with `collections.OrderedDict`.